portality/templates/dashboard/index.html

```diff
@@ -1,41 +1,95 @@
 {% extends "layouts/dashboard_base.html" %}
 {# ~~Dashboard:Page~~ #}
 {% set TODOS = {
-        constants.TODO_MANED_STALLED: "Chase <strong>stalled</strong> application",
-        constants.TODO_MANED_FOLLOW_UP_OLD: "Follow up with <strong>old<strong> application",
-        constants.TODO_MANED_READY: "Make a decision on <strong>ready<strong> application",
-        constants.TODO_MANED_COMPLETED: "Review <strong>completed<strong> application",
-        constants.TODO_MANED_ASSIGN_PENDING: "Assign <strong>pending<strong> application"
+        constants.TODO_MANED_STALLED: "Chase a <strong>stalled</strong> application",
+        constants.TODO_MANED_FOLLOW_UP_OLD: "Follow up with an <strong>old<strong> application",
+        constants.TODO_MANED_READY: "Make a decision on a <strong>ready<strong> application",
+        constants.TODO_MANED_COMPLETED: "Review a <strong>completed<strong> application",
+        constants.TODO_MANED_ASSIGN_PENDING: "Assign a <strong>pending<strong> application"
     }
 %}
 
+
+
 {% block content %}
 <main>
-
   <h1>Hi, {{ current_user.name }}!</h1>
 
   <section>
-    <h2 class="label">Here are your priorities:</h2>
-    <ol class="unstyled-list">
+    <h2 class="type-04">Here are your priorities:</h2>
+    <ol class="unstyled-list todo-list">
       {% for todo in todos %}
       {# TODO only show tasks for this user’s groups #}
       {# TODO integrated priority in list display
       {{ todo.boost }}
        #}
-      <li class="card card--compact">
+       {% set app_url =  url_for('admin.application', application_id=todo.object_id) %}
+      <li
+        class="card card--compact todo-list__item"
+        data-app-url="{{ app_url }}"
+        {# TODO there’s probably a better way to do this... #}
```

Comment:
oh yes, for sure :)  I'll push a change in a moment...

Reply:
I've pushed this change now, so if you pull the branch again you will get my approach

Reply:
Looks great! Thank you! I was trying to find a way to do that but really couldn’t figure it out :)